pywps/templates/1.0.0/describe/literal.xml

```diff
@@ -32,6 +32,6 @@
                     {% endfor %}
                 </ows:AllowedValues>
                 {% endif %}
-                {% if put.data %}
+                {% if put.data is not none %}
```

Comment:
shouldn't this be `is not None`?

Reply:
Yes, that's what I thought first also. Jinja uses test functions, not straight python evaluation. See: https://github.com/pallets/jinja/issues/520

Reply:
ok.